coop_membership/views/res_config_view.xml

```diff
@@ -92,6 +92,16 @@
                                 </div>
                             </div>
                         </div>
+                        <div class="col-12 col-lg-6 o_setting_box">
+                            <div class="o_setting_right_pane">
+                                <div class="content">
+                                    <h2>Office Timing</h2>
```

Comment:
```suggestion
                                    <h2>Members Office Open Hours</h2>
```

It'd be good to add a note or describption here so that it's easier to understand the impact of this field.